doc/api/util.md

```diff
@@ -2008,11 +2013,20 @@ changes:
     the built-in behavior, from adding additional checks through to reprocessing
     the tokens in different ways.
     **Default:** `false`.
+  * `help` {string} General help text to display at the beginning of help output.
+  * `addHelpOption` {boolean} Whether to automatically add a help option to the
```

Comment:
what happens if `addHelpOption` is true and `help` is not provided? is there a default help text?